src/panels/lovelace/components/hui-generic-entity-row.js

```diff
@@ -0,0 +1,89 @@
+import { html } from '@polymer/polymer/lib/utils/html-tag.js';
+import { PolymerElement } from '@polymer/polymer/polymer-element.js';
+
+import '../../../components/entity/state-badge.js';
+import '../../../components/ha-relative-time.js';
+
+import computeStateName from '../../../common/entity/compute_state_name.js';
+
+import EventsMixin from '../../../mixins/events-mixin.js';
+
+/*
+ * @appliesMixin EventsMixin
+ */
+class HuiGenericEntityRow extends EventsMixin(PolymerElement) {
+  static get template() {
+    return html`
+      <style>
+        :host {
+          cursor: pointer;
+          display: flex;
+        }
+        .flex {
+          margin-left: 16px;
+          flex: 1;
+          display: flex;
+          align-items: center;
+          justify-content: space-between;
+        }
+        .secondary,
+        ha-relative-time {
+          display: block;
+          color: var(--secondary-text-color);
+        }
+      </style>
+      <template is="dom-if" if="[[_stateObj]]">
```

Comment:
If we hide everything when there is no stateObj, can we instead show a light yellow div saying "State X not found"?